images/kubekins-e2e/Dockerfile

```diff
@@ -37,9 +37,11 @@ RUN mkdir -p /go/src/k8s.io/kubernetes \
 # - graphviz package for graphing profiles
 # - bc for shell to junit
 # - rpm for building RPMs with Bazel
+# - gcc-aarch64-linux-gnu for cross-compile arm64 binaries
```

Comment:
nit: let's keep the same order in comment and dependencies.

Reply:
Done. Updated based on suggestion.